code/game/machinery/washing_machine.dm

```diff
@@ -377,6 +383,18 @@ GLOBAL_LIST_INIT(dye_registry, list(
 			if(istype(L, /mob/living/basic/pet))
 				L.forceMove(src)
 				update_appearance()
+			if(istype(L, /mob/living/carbon/human))
+				if(user.grab_state < GRAB_AGGRESSIVE)
+					to_chat(user, span_warning("You need a tighter grip!"))
+					return
+
+				L.visible_message(span_danger("[user] is trying to force [L] into the washing machine!"))
+				log_game("[key_name_and_tag(user)] is forcing [key_name_and_tag(L)] into a washing machine")
+				if(!do_after(user, 3 SECONDS, target = src, timed_action_flags = IGNORE_HELD_ITEM))
```

Comment:
I think this means that if you start it and the victim escapes and runs away, they'll still be put in? Forget exactly the best way to make sure it cancels though we have a lot of do_after procs